Delete PRT using company portal (#111796)
Azure/azure-content: articles/active-directory/devices/troubleshoot-mac-sso-extension-plugin.md

|**Managed**| Indicates that device is under management. |
|**Join Type**| macOS and iOS, if registered, can only be of type: **Azure AD Registered**. |

> #### Delete PRT Using Company Portal

Reviewer: ```suggestion
#### Delete PRT using Company Portal
```

The following steps can be used to remove a PRT of the device with the Company Portal:

> 1. From the macOS device, click on the spotlight icon

Reviewer: ```suggestion
1. From the macOS device, select the spotlight icon.
```

> 1. When the Spotlight Search appears, type Company Portal and hit return.

Reviewer: ```suggestion
1. When the Spotlight Search appears, type "Company Portal" and press Return.
```

> 1. When the Company Portal page loads, click on the account logged in on the top right hand corner

Reviewer: ```suggestion
1. When the Company Portal page loads, select the account logged in at the top right corner.
```

> 1. On this page, click Remove account from this device button

Reviewer: ```suggestion
1. On this page, select the **Remove account from this device** button.
```

> 1. On the keychain access window, refresh the search and you will be able to validate the PRT has been remove

Reviewer: ```suggestion
1. On the keychain access window, refresh the search and validate that the PRT has been removed.
```
